### DISCUSSION TITLE
webpack5升级rsbuild后，主包打包更大

### DISCUSSION THREAD

**Discussion by shiwenna** [web-infra-dev/rsbuild#4118] [Q&A]
webpack5打包后，主包main.js文件为1200K。升级Rsbuild后默认配置打包index.js主文件3200K，使用split-by-module模式index.js
也在2000K大小，想问一下有什么方式能减少主包大小

**Comment by chenjiahan** [comment]
See: https://rsbuild.dev/guide/optimization/optimize-bundle

**Comment by cieme** [comment]
I'm 400KB bigger
